src/templates/Editorial/index.js

```diff
@@ -0,0 +1,392 @@
+              }),
+              editorModule: 'block',
+              editorOptions: {
+                type: 'QUOTE',
+                insertButtonText: 'Quote'
+              },
+              rules: [
+                {
+                  matchMdast: matchZone('FIGURE'),
+                  component: PullQuoteFigure,
+                  editorModule: 'figure',
+                  editorOptions: {
+                    type: 'QUOTEFIGURE',
+                    afterType: 'QUOTEP'
+                  },
+                  rules: [
+                    {
+                      matchMdast: matchImageParagraph,
+                      component: FigureImage,
+                      props: node => ({
+                        data: {
+                          src: node.children[0].url,
+                          alt: node.children[0].alt
+                        }
+                      }),
+                      editorModule: 'figureImage',
+                      isVoid: true
+                    },
+                    {
+                      matchMdast: matchParagraph,
+                      component: FigureByline,
+                      editorModule: 'paragraph',
+                      editorOptions: {
+                        type: 'FIGURECAPTION',
+                        placeholder: 'Credit'
+                      },
+                      rules: [
+                        {
+                          matchMdast: matchType('emphasis'),
+                          component: FigureByline,
+                          // ToDo: inline module with placeholder
+                          editorOptions: {
+                            placeholder: 'Credit'
+                          }
+                        },
+                        link,
+                        br
+                      ]
+                    }
+                  ]
+                },
+                {
+                  matchMdast: (node, index, parent) => (
+                    matchParagraph(node) &&
+                    (index === 0 || !matchLast(node, index, parent))
+                  ),
+                  component: PullQuoteText,
+                  editorModule: 'paragraph',
+                  editorOptions: {
+                    type: 'QUOTEP',
+                    placeholder: 'Zitat'
+                  },
+                  rules: [
+                    br,
+                    link
+                  ]
+                },
+                {
+                  matchMdast: (node, index, parent) => matchParagraph(node) && matchLast(node, index, parent),
+                  component: PullQuoteSource,
+                  editorModule: 'paragraph',
+                  editorOptions: {
+                    type: 'QUOTECITE',
+                    placeholder: 'Quellenangabe / Autor'
+                  },
+                  rules: [
+                    br,
+                    link
+                  ]
```

Comment:
No action required now, but just for the record:
As a left-wing, this code is veering way too much to the right IMHO ;) 
We might wanna consider breaking things down into smaller objects, and composing the main schema from these.

Reply:
Word